elastalert/elastalert.py

```diff
@@ -1056,6 +1056,7 @@ def init_rule(self, new_rule, new=True):
                                      args=[new_rule],
                                      seconds=new_rule['run_every'].total_seconds(),
                                      id=new_rule['name'],
+                                     name="Rule: %s" % (new_rule['name']),
```

Comment:
The name is used in log messages.  Without the name, the messages currently look like:

```
WARNING:apscheduler.scheduler:Execution of job "ElastAlerter.handle_rule_execution (trigger: interval[0:02:30], next run at: 2021-06-29 13:21:11 UTC)" skipped: maximum number of running instances reached (1)
```

But with the name they look like:

```
WARNING:apscheduler.scheduler:Execution of job "Rule: Errors (trigger: interval[0:00:10], next run at: 2021-06-29 09:28:25 EDT)" skipped: maximum number of running instances reached (1)
```